### PR TITLE
COG-1234 Actions/Saga/Reducer for audit events

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1234

#### Brief feature description
- Added Redux actions, saga, reducer to fetch audit events for user list page

### CODE REVIEW

File: app/javascript/_services/users/user.service.js [hunk 1]
```diff
@@ -17,6 +17,19 @@ class UserService {
     return ApiService.get(`/user_list?q=${q}`).then(({ data }) => data)
   }
 
+  static auditEvents({ query = [], sort = [], size = 10, from = 0 }) {
+    const q = encodeURIComponent(
+      JSON.stringify({
```
Comment (kelfink) [inline app/javascript/_services/users/user.service.js]: can this call fit onto one line?

Comment (RamuKammagani-osi) [reply app/javascript/_services/users/user.service.js]: I guess so, let me try and update it

Comment (RamuKammagani-osi) [reply app/javascript/_services/users/user.service.js]: Updated!

File: app/javascript/reducers/auditEventsReducers.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import * as actionTypes from '../actions/actionTypes'
+
+const initialValue = {
+  sort: [
+    // {
```
[the remark this answers is not in the document]
Comment (RamuKammagani-osi) [reply app/javascript/reducers/auditEventsReducers.js]: Updated!

**Pull request merged**